Makefile

```
@@ -21,7 +21,7 @@ style:
 # this only checks for style & pyright, makes no code changes
 lint:
 	$(PYTHON) -m isort -c --diff $(dirs)
-	$(PYTHON) -m yapf -dr $(dirs)
+	$(PYTHON) -m yapf -ri $(dirs)
```

```suggestion
	$(PYTHON) -m yapf -dr $(dirs)
```

reverting this change -- `make lint` should not be in-place.